### ISSUE TITLE
SUnit browser and SUnit tab in Project browser missing inherited test cases

### ISSUE THREAD

**Issue by dalehenrich** [ericwinger/Jade#211]
In the [FileSysteGs project](https://github.com/GemTalk/FileSystemGs), the class **FileHandleTest** inherits a number of test cases from **FileSystemHandleTest**:
```Smalltalk
FileHandleTest suite tests collect: [:each | each printString]
```
returns:
```
anOrderedCollection(
 'FileHandleTest>>#testAt', 
'FileHandleTest>>#testAtPut', 
'FileHandleTest>>#testAtPutBinaryAscii',
 'FileHandleTest>>#testAtWriteBinaryAscii',
 'FileHandleTest>>#testClose', 
'FileHandleTest>>#testCreatedOpen', 
'FileHandleTest>>#testEnsureClosed',
 'FileHandleTest>>#testIO', 
'FileHandleTest>>#testReadBufferTooLarge', 
'FileHandleTest>>#testReadOnly', 
'FileHandleTest>>#testReference', 
'FileHandleTest>>#testSizeAfterGrow', 
'FileHandleTest>>#testSizeNoGrow', 
'FileHandleTest>>#testTruncate', 
'FileHandleTest>>#testWriteStream'
)
```
But both the SUnit browser and Project browser show an empty list ... I assume that Jadeite is using #testSelectors instead of #suite to determine the list of tests for a test case ...

**Comment by dalehenrich** [comment]
`Run Tests` from the class in the project browser, gives you the list of test cases but it is using the wrong class name ...:
```
FileSystemHandleTest debug: #'testAt'.
FileSystemHandleTest debug: #'testAtPut'.
FileSystemHandleTest debug: #'testAtPutBinaryAscii'.
FileSystemHandleTest debug: #'testAtWriteBinaryAscii'.
FileSystemHandleTest debug: #'testClose'.
FileSystemHandleTest debug: #'testEnsureClosed'.
FileSystemHandleTest debug: #'testIO'.
FileSystemHandleTest debug: #'testReadBufferTooLarge'.
FileSystemHandleTest debug: #'testReadOnly'.
FileSystemHandleTest debug: #'testSizeAfterGrow'.
FileSystemHandleTest debug: #'testSizeNoGrow'.
FileSystemHandleTest debug: #'testTruncate'.
FileSystemHandleTest debug: #'testWriteStream'.
```

**Comment by dalehenrich** [comment]
The FileSystemTests are on the issue_361 branch ... I think this is one is right up there with `need to fix soonish`, since the sunit browser/pane cannot be used to run tests for the tests cases --- and these are valid test case structures

**Comment by ericwinger** [comment]
A fix will be in Oscar 3.0.51.
